scss/_patterns_grid.scss

```diff
@@ -87,6 +102,12 @@
     }
   }
 
+  %span-full-grid-on-tablet-new {
+    @media (min-width: $threshold-4-small-4-med-col) {
+      grid-column: auto / span $grid-columns-medium-new;
+    }
+  }
+
   @for $i from 1 through $grid-columns-small {
     .#{$grid-small-col-prefix}#{$i} {
       @extend %span-full-grid-on-tablet;
```

Comment:
May need to extend the span full 4-col-tablet-grid placeholder here as well, for consistency

Reply:
Fixed